src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -1,5 +1,13 @@
 package org.usfirst.frc4904.robot;
 
+
+import org.usfirst.frc4904.robot.humaninterface.HumanInterfaceConfig;
```

Comment:
Make sure to update standard to the newest `master` HEAD.
```bash
$ cd src/org/usfirst/frc4904/standard/
$ git checkout master
$ git pull
$ cd .. # this will take you back to the domain of 2018-Code
$ git status # this should say "new commits on standard"
$ git add standard
$ git commit {etc}
```